contrib/segyconverter/convert_segy.py

```diff
@@ -0,0 +1,140 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+"""
+Utility Script to convert segy files to blocks of numpy arrays and save to individual npy files
+"""
+
+import os
+import timeit
+import argparse
+import numpy as np
+import utils.segyextract as segyextract
+import utils.dataprep as dataprep
+import json
+
+K = 12
+MIN_VAL = 0
+MAX_VAL = 1
+
+def _filter_data(output_dir, stddev_file, k, min_range, max_range, clip, normalize):
```

Comment:
Why does this start with _? Is it a private method?

Reply:
yes. private method that I didn't intend to be used externally. Does your code base use a different standard?

Reply:
@maxkazmsft Does your code base use a different standard?

Reply:
it's a method inside a script, hence it's public to the script and not private